src/db/dal/user.ts

```diff
@@ -1,24 +1,130 @@
+import AWS from 'aws-sdk';
 import createHttpError from 'http-errors';
 import { StatusCodes } from 'http-status-codes';
+import { Op, Order } from 'sequelize';
+import { uuid } from 'uuidv4';
 
+import { profileImageBucket } from '../../env';
 import { UserValidator } from '../../utils/userValidator';
 import UserModel, { IUserInput, IUserOuput } from '../models/User';
 
-const sanitizeInputPayload = (payload: IUserInput) => ({
-    ...payload,
-    id: undefined,
-    keycloak_id: undefined,
-    completed_registration: undefined,
-    creation_date: undefined,
-});
+let S3Client;
```

Comment:
catch potential error from AWS sdk if no config is setted in env
should we log it on catch ? ...